debian/postinst

```diff
@@ -14,6 +14,9 @@ do_configure() {
       echo "Regenerating configuration, this might take a while..."
       yunohost service regen-conf --output-as none
 
+      echo "Launching migrations.."
+      yunohost migrations migrate
```

Comment:
I hope this is the good place to do it, I did it in the same place than the regen-conf assuming this was the correct location.